extensions/ql-vscode/src/remote-queries/variant-analysis-view.ts

```diff
@@ -130,4 +134,29 @@ export class VariantAnalysisView extends AbstractWebview<ToVariantAnalysisMessag
       void showAndLogWarningMessage(`Could not open file: ${variantAnalysis.query.filePath}`);
     }
   }
+
+  private async openQueryText(): Promise<void> {
+    const variantAnalysis = await this.manager.getVariantAnalysis(this.variantAnalysisId);
+    if (!variantAnalysis) {
+      void showAndLogWarningMessage('Could not open variant analysis query text');
+      return;
+    }
+
+    const filename = variantAnalysis.query.filePath;
+
+    try {
+      const params = new URLSearchParams({
+        variantAnalysisId: variantAnalysis.id.toString(),
+      });
+      const uri = Uri.from({
+        scheme: 'codeql-variant-analysis',
+        path: filename,
+        query: params.toString(),
+      });
+      const doc = await workspace.openTextDocument(uri);
+      await Window.showTextDocument(doc, { preview: false });
+    } catch (error) {
+      void showAndLogWarningMessage('Could not open query text');
```

Comment:
Just to make the error match the other one earlier in this function. It could be useful having the errors be different so we can tell them apart in bug reports, but if that's the plan let's make them more descriptive, so for example the earlier error would be something like "Could not open variant analysis query text. Variant analysis not found."
```suggestion
      void showAndLogWarningMessage('Could not open variant analysis query text');
```